models/graph_convolutions/appnp.py

```diff
@@ -0,0 +1,35 @@
+# labels: test_group::mlagility name::appnp author::graph_convolutions
```

Comment:
Move all of these to `graph_convolutions/skip/*`

Reply:
moved!